src/components/exchange/ExchangeHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { SheetHandle } from '../sheet';
+import { Heading, Inset, Stack } from '@rainbow-me/design-system';
+
+interface ExchangeHeaderProps {
+  testID: string;
+  title: string;
+}
+
+export default function ExchangeHeader({ testID, title }: ExchangeHeaderProps) {
+  return (
+    <Inset space="6px">
+      <Stack alignHorizontal="center" space="8px">
```

Comment:
wondering where this `8px` coming from, we only have 6 in figma 
![image](https://user-images.githubusercontent.com/12115171/186047774-838a020d-935b-4bfd-90dd-feabc3f86a10.png)

Reply:
https://www.figma.com/file/FNzceJrmBJMUPgSj0oVXm9/Swap-Aggregator-V1?node-id=2885%3A51380

Reply:
@estebanmino – think it accounts for the line height. That design doesn't have capsize on the text node.

Reply:
ok new designs are showing 10 now https://www.figma.com/file/SFfYzRoHlapEecS0LTLwhQ/Swap-Aggregator-V2?node-id=53%3A21958 this is with capsize